src/main/java/prediction/LanguageMatcher.java

```diff
@@ -0,0 +1,24 @@
+package prediction;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+
+
+public class LanguageMatcher {
+    private String name;
```

Comment:
the attribute's name is a kinda ambiguous. If it's the book's name, better specify it like so.